ng2-components/ng2-activiti-tasklist/src/components/activiti-people-search.component.ts

```diff
@@ -85,17 +84,38 @@ export class ActivitiPeopleSearch implements OnInit, AfterViewInit {
         return isUpgraded;
     }
 
-    onRowClick(userClicked: User) {
-        this.onRowClicked.emit(userClicked);
-        this.userList = this.userList.filter((user) => {
+    onRowClick(user: User) {
+        this.selectedUser = user;
+    }
+
+    closeSearchList() {
+        this.success.emit();
```

Comment:
It is not clear to me why the closing of the search emits a success event. Maybe you intend to emit a different event name like "closeSearch" or "endSearch" ?
I think that success here is ambiguous.